plugin/trino-delta-lake/src/main/java/io/trino/plugin/deltalake/transactionlog/TransactionLogAccess.java

```diff
@@ -226,14 +227,39 @@ public MetadataEntry getMetadataEntry(TableSnapshot tableSnapshot, ConnectorSess
     @Deprecated
```

Comment:
Should this deprecation be associated with a code comment informing the why or the direction we plan to evolve in?